app/code/Magento/Eav/Model/ResourceModel/Entity/Attribute/Group/Collection.php

```diff
@@ -40,6 +42,20 @@ public function setAttributeSetFilter($setId)
         return $this;
     }
 
+    /**
+     * Set Attribute Set Condition Type Set Filter
+     *
+     * @param string $setId
+     * @param string $conditionType
+     * @return $this
+     */
+    public function setAttributeSetConditionSetFilter($setId, $conditionType = 'eq')
```

Comment:
Unfortunately, we are not allowed to add new public methods to classes marked as @api according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
Thanks @sidolov, this change were made.